pkg/plugins/handlers.go

```diff
@@ -26,6 +26,79 @@ func NewPluginsHandler(client *http.Client, token string, pluginsEndpointMap map
 	}
 }
 
+func (p *PluginsHandler) HandleLocales(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		w.Header().Set("Allow", "GET")
+		serverutils.SendResponse(w, http.StatusMethodNotAllowed, serverutils.ApiError{Err: "Method unsupported, the only supported methods is GET"})
+		return
+	}
+
+	lang, pluginLocaleFile, err := parseLangAndPluginLocaleFile(r.URL.Path)
+	if err != nil {
+		errMsg := err.Error()
+		klog.Error(errMsg)
+		serverutils.SendResponse(w, http.StatusBadRequest, serverutils.ApiError{Err: errMsg})
+		return
+	}
+	redirectTo := fmt.Sprintf("static/locales/%s/%s", lang, pluginLocaleFile)
+	pluginName := strings.SplitN(pluginLocaleFile, ".", 2)[0]
+	for _, enabledPluginName := range p.PluginsEndpointMap {
+		// if the request if for enabled plugin locale, then proxy to it's service
+		if enabledPluginName == pluginName {
+			pluginServiceEndpoint, err := p.getServiceRequestURL(pluginName)
+			if err != nil {
+				errMsg := err.Error()
+				klog.Error(errMsg)
+				serverutils.SendResponse(w, http.StatusBadRequest, serverutils.ApiError{Err: errMsg})
+				return
+			}
+			redirectTo = fmt.Sprintf("%s/%s/%s", pluginServiceEndpoint, lang, pluginLocaleFile)
+		}
+		klog.Errorf(redirectTo)
+	}
+
+	resp, err := p.Client.Get(redirectTo)
```

Comment:
Is this actually making an HTTP request for static files? Can we serve static files directly instead using `http.ServeFile`?

Reply:
we should